pkg/v7/resource/chartconfig/desired.go

```diff
@@ -40,6 +40,7 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interf
 		r.newKubeStateMetricsChartConfig,
 		r.newNetExporterChartConfig,
 		r.newNodeExporterChartConfig,
+		r.newCoreDNSChartConfig,
```

Comment:
Can we keep the generators in alpha order.

Reply:
Sure, done.